backend/src/api/routes/me.py

```diff
@@ -0,0 +1,41 @@
+from typing import Annotated
+
+from fastapi import APIRouter
+
+from src.api.dependencies import LoggedInUser
+from src.auth import User
+from src.dependencies import Db
+from src.models import Idea, IdeaPublic, IdeasPublic, UserEditPatch
+
+router = APIRouter(prefix="/me")
+
+
+@router.get("/")
+async def get_me(current_user: Annotated[User, LoggedInUser]):
+    return current_user
```

Comment:
```suggestion
    return UserMe(**current_user.model_dump())
```
After this modification, the `"/me"` route in `main.py` can be completely removed.